emoji-ui/src/components/SlackTokenInput.js

```diff
@@ -49,6 +49,12 @@ class SlackTokenInput extends Component {
           />
         </label>
 
+        <small className="token-input-help-text">
+          API keys can be found under window.TS.boot_data.api_token
+          while inspecting the "customize slack" webpage of a slack
+          instance. They should start with xoxs-*.
+        </small>
+
         <input className="token-input-submit" type="submit" value="Save" />
```

Comment:
we'll probably want to save these in redux and pass them with the get/post calls (this is a note for later)

Reply:
yep right now the save button doesn't do anything